mysql-test/include/test_outfile.inc

```diff
@@ -0,0 +1 @@
+eval select "Outfile OK" into outfile "$MYSQLTEST_VARDIR/tmp/outfile.test";
```

Comment:
I believe this should be `$MYSQL_TMP_DIR` instead of `$MYSQLTEST_VARDIR/tmp`. There is a difference between them then MTR is run with --parallel=n
```suggestion
eval select "Outfile OK" into outfile "$MYSQL_TMP_DIR/outfile.test";
```